src/btrfs.cpp

```diff
@@ -0,0 +1,154 @@
+/****************************************************************************************
+**
+** Copyright (C) 2015 Jolla Ltd.
+** Contact: Martin Grimme <martin.grimme@gmail.com>
+** All rights reserved.
+**
+** This file is part of the btrfs-balancer package.
+**
+** You may use this file under the terms of the GNU Lesser General
+** Public License version 2.1 as published by the Free Software Foundation
+** and appearing in the file license.lgpl included in the packaging
+** of this file.
+**
+** This library is free software; you can redistribute it and/or
+** modify it under the terms of the GNU Lesser General Public
+** License version 2.1 as published by the Free Software Foundation
+** and appearing in the file license.lgpl included in the packaging
+** of this file.
+**
+** This library is distributed in the hope that it will be useful,
+** but WITHOUT ANY WARRANTY; without even the implied warranty of
+** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+** Lesser General Public License for more details.
+**
+****************************************************************************************/
+
+
+#include "btrfs.h"
+
+#include <QLocale>
+#include <QRegExp>
+#include <QDebug>
+
+namespace
+{
+// path of the btrfs tool
+const QString BTRFS_PATH("/usr/sbin/btrfs");
+
+// default root device to use
+const QString ROOT_DEVICE("/dev/mmcblk0p28");
```

Comment:
Yes, we should make that configurable. Either the RPM should generate its own `-sbj-config` RPM sub-package with config files that it reads (and ship the path there), or re-use some existing ones, e.g. [partition-sbj.conf](https://github.com/sailfishos/sailfish-snapshot/blob/master/data/partition-sbj.conf) from sailfish-snapshot (the [RPM](https://github.com/sailfishos/sailfish-snapshot/blob/master/rpm/sailfish-snapshot.spec) is called `sailfish-snapshot-sbj-config`)